packages/jss-keyframes-helper/README.md

```diff
@@ -0,0 +1,80 @@
+# jss-keyframes-helper
+
+Helper for generating keyframes with a unique id in jss
+
+## Installation
+
+> yarn add jss-keyframes-helper
+
+## Usage
+
+```js
+import keyframes from 'jss-keyframes-helper'
+
+const animationName = keyframes({
+  from: {transform: 'scale(1)'},
+  to: {transform: 'scale(0)'}
+})
+```
+
+The keyframes function returns the name of the animation.
+
+## Options
+
+The keyframes function accepts a second object for some configuration.
+
+```js
+type Options = {
+  // Jss instance to be used
+  // It will generate a separate stylesheet for every instance
+  jss: Jss
+  // A name to better identify the animation
+  // Example: { name: 'button' } would return button-1
+  // This option will default to animation
+  name: string
+  // A StyleSheet to be used for adding the keyframes to instead of the created one.
+  // This will not add the stylesheet to the internally used SheetsRegistry of this package.
+  // You will need to add it to your own SheetsRegistry when doing SSR.
+  sheet: StyleSheet,
+}
+```
+
+## SSR with React
+
+Because we may create multiple stylesheets for different jss instances, we add those to an internal SheetsRegistry which you can import.
+Keyframes should be static, so you don't need to recreate them for every requests and reuse them between requests.
```

Comment:
What if they want to use fn values inside of them? Not sure if it even works.

Reply:
Ohh yeah, function values do not work with this package, though I feel like you usually don't need function values inside keyframes.